src/writer/vault.py

```
@@ -0,0 +1,53 @@
+import os
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)